doc/nrf/config_and_build/bootloaders_and_dfu/bootloader.rst

```diff
@@ -103,7 +123,7 @@ The :ref:`bootloader capabilities table <app_bootloaders_support_table>` indicat
 
 .. _upgradable_bootloader:
 
-Second-stage Upgradable bootloader
+Second-stage upgradable bootloader
```

Comment:
This now looks like a sub-header of "Static partition requirement for DFU".
And I beleive that it is its own section right?

Reply:
Fixed as mentioned above.